docs/source/api/v4/async_status.rst

```diff
@@ -0,0 +1,52 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-async_status:
+
+****************
+``async_status``
+****************
+
+``GET``
+=======
+Returns a status update for an asynchronous task.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+No parameters available
+
+
+Response Structure
+------------------
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Content-Type: application/json
+
+	{ "response":
+		{
+			"id":1,
+			"status":"PENDING",
+			"start_time":"2021-02-18T17:13:56.352261Z",
+			"end_time":null,
+			"message":"Async job has started."
+		}
+	}
```

Comment:
How do I know which job is mine, based on this? Since you're exposing these database IDs through the API anyway, maybe you could add a query or path parameter so that the returned Location/Alert can be `async_status/{{ID}}` or `async_status?id={{ID}}`?